### ISSUE TITLE
Padding mode support?

### ISSUE THREAD

**Issue by fangxiao-dev** [kevinzakka/spatial_transformer_network#30]
Thanks for the work. I notice that it's by default *zero-padded*. I wonder if there's any way to achieve a different padding mode? For example, the `torch.nn.grid_sample()` supports "border" padding.
